### ISSUE TITLE
The pecl stats page could be improved to encourage developers of new extensions

### ISSUE THREAD

**Issue by qwerty199369** [php/web-pecl#103]
The pecl stats page (https://pecl.php.net/package-stats.php) should add a time range dimension. Because some of the most downloaded extensions seem to have stopped maintenance, but are still ranked at the top (due to the large number of historical downloads). The current mechanism is not friendly to new and excellent extensions, and is not conducive to encouraging developers and maintainers of popular extensions.

**Comment by cmb69** [comment]
I generally agree, but given that [PIE](https://github.com/php/pie) is supposed to replace PECL, I'm not sure it's worth spending time on improving the PECL Website.

**Comment by Girgias** [comment]
I would agree that given PIE should soon TM have an initial alpha release and that you can already use it to install some extensions (see: https://packagist.org/extensions) updating PECL seems a bit pointless

**Comment by qwerty199369** [comment]
Looking forward to the official release of PIE and thank you for your super good work!

**Comment by derickr** [comment]
PIE has been released a while ago! I'm going to close this down, as we have no intention of improving the PECL Website (beyond shutting it down at some point).
